src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -138,9 +138,12 @@ public String getScript() {
         String cmd;
         FilePath logFile = c.getLogFile(ws);
         FilePath resultFile = c.getResultFile(ws);
+        if (resultFile.exists()) {
+            resultFile.delete();  // Maybe overly cautious, but better safe than sorry, similarly we should make sure no duplicate logfile?
```

Comment:
Well, the whole control directory should not exist before we start, or should be empty.